release-1.8/release_notes_draft.md

```diff
@@ -291,13 +292,13 @@ kind.
  must set the spec.selector in their manifests, and the creation of an object
  with a spec.selector that does not match the labels in its spec.template is
  considered to be invalid.
- - As none of the controllers in the workloads API handle selector mutation in
- a consistent way, selector mutation is disabled in for all kinds in the
+ - Because the controllers in the workloads API do not handle selector mutation in
+ a consistent way, selector mutation is disabled for all kinds in the
  app/v1beta2 group version. This restriction may be lifted in the future, but
- it is likely that that selectors will remain immutable after GA promotion.
- Users that have any code that depends on mutable selectors may continue to use
+ it is likely that that selectors will remain immutable after v1 (GA) promotion.
```

Comment:
I would only add GA, maybe, @calebamiles has thoughts on this usage.

Reply:
I left v1 and removed GA here because API